app/code/Magento/Contact/Test/Unit/Controller/Index/PostTest.php

```diff
@@ -207,7 +207,7 @@ private function stubRequestPostData($post): void
         $this->requestStub->method('getParams')->willReturn($post);
         $this->requestStub->method('getParam')->willReturnCallback(
             function ($key) use ($post) {
-                return $post[$key];
+                return $post[$key] ?? '';
```

Comment:
It seems that we don't need `if` statement on L148. `exceptionExpected` always set to true from data provider. Could you please remove it ?

Reply:
@xmav, I added changed by your recommendation, please check if every think is well. 

Thank you